qpmodel/Plan.cs

```diff
@@ -344,7 +344,7 @@ LogicNode transformOneFrom(TableRef tab)
                         break;
                     case QueryRef qref:
                         var plan = qref.query_.CreatePlan();
-                        if (qref is FromQueryRef && queryOpt_.optimize_.remove_from_)
+                        if (qref is FromQueryRef && queryOpt_.optimize_.remove_from_&&queryOpt_.optimize_.enable_subquery_unnest_)
```

Comment:
enable_subquery_unnest_ and remove_from are not independent, without this condition, the from query will not add to **subqueries**. And the **subquris** will not have physic plan. see CopyOutOptimalPlan
``` 
            foreach (var v in subqueries)
                CopyOutOptimalPlan(v.query_, !select.SubqueryIsWithMainQuery(v));
```

Reply:
We may want to decouple them: 
- remove_from is used to collapse multiple FROM subquery, which is not correlated. Say "select * from (select * from a) b;" becomes "select * from a". You shall see no more LogicFromQuery in plan. 
- subquery_unnest is to decorrelate correlated subqueries.

So logically they are independent.

Reply:
OK, I'll check it in detail latter. The PR is just a workround.